frontend/packages/policy-editor/src/components/PolicySummary/PolicyRuleSummary/PolicyRuleSummary.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import type { PolicyRuleCard } from '../../../types';
+import { useTranslation } from 'react-i18next';
+import { usePolicyEditorContext } from '../../../contexts/PolicyEditorContext';
+import { StudioTable, StudioTag } from '@studio/components';
+import { getSubjectDisplayName, getSubResourceDisplayText } from '../../../utils/AppPolicyUtils';
+
+export type PolicyRuleSummaryProps = {
+  policyRule: PolicyRuleCard;
+  showErrors: boolean;
+  ruleIndex?: number;
+};
+
+export const PolicyRuleSummary = ({ policyRule }: PolicyRuleSummaryProps): React.ReactNode => {
+  const { t } = useTranslation();
+  const { usageType, subjects } = usePolicyEditorContext();
+
+  const getActionsDisplayNames = (actionList: string[]): string => {
+    return actionList
+      .map((a) => {
+        return t(`policy_editor.action_${a}`);
+      })
+      .join(', ');
+  };
```

Comment:
Based on the existing pattern of utility functions in the `ArrayUtils` class, should we consider adding the arrayToString function there? 🤔

```suggestion
  const getActionsDisplayNames = (actionList: string[]): string => {
     const translations: string[] = actionList.map((action) => t(`policy_editor.action_${action}`));
     return ArrayUtils.toString(translations);
  }
```